src/repairchain/strategies/generation/llm/llm.py

```diff
@@ -57,12 +57,12 @@ def _simple_call_llm(self, messages: MessagesIterable) -> str | None:
                     return None
                 return llm_output
 
-            except openai.APIConnectionError as e:
-                logger.warning(f"Connection error: {e}. Retrying {attempt + 1}/{retry_attempts}...")
-                time.sleep(Util.short_sleep)  # brief wait before retrying
             except openai.APITimeoutError as e:
                 logger.warning(f"API timeout error: {e}. Retrying {attempt + 1}/{retry_attempts}...")
                 time.sleep(Util.short_sleep)  # brief wait before retrying
+            except openai.APIConnectionError as e:
```

Comment:
why the move? is `APITimeoutError` a subclass of `APIConnectionError`?

Reply:
I think it is because I was getting unreachable code. That is why I moved it but then maybe we do not want both.

Reply:
what was reporting the code as unreachable?

Reply:
Just VSCode ....

The potential errors are listed here: https://platform.openai.com/docs/guides/error-codes/api-errors
No information about being subclasses.